packages/core/contracts/oracle/interfaces/SkinnyOptimisticOracleInterface.sol

```diff
@@ -0,0 +1,240 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "../interfaces/OptimisticOracleInterface.sol";
+
+/**
+ * @title Interface for the gas-cost-reduced version of the OptimisticOracle.
+ * @dev Interface used by financial contracts to interact with the Oracle. Voters will use a different interface.
+ */
+abstract contract SkinnyOptimisticOracleInterface {
+    // Struct representing a price request. Note that this differs from the OptimisticOracleInterface's Request struct
+    // in that refundOnDispute is removed.
+    struct Request {
+        address proposer; // Address of the proposer.
+        address disputer; // Address of the disputer.
+        IERC20 currency; // ERC20 token used to pay rewards and fees.
+        bool settled; // True if the request is settled.
+        int256 proposedPrice; // Price that the proposer submitted.
+        int256 resolvedPrice; // Price resolved once the request is settled.
+        uint256 expirationTime; // Time at which the request auto-settles without a dispute.
```

Comment:
If we do change this to uint32, we should move it next to the boolean above, so it can be packed tightly and save some bytes.

Reply:
Would we also need to change the order of function params to have uint32's next to each other?

Reply:
I looked into the amount of gas this would save and it's not a huge amount unless you're actually storing the struct. We can take a look at it later, but I don't think it's going to impact the bottom line too much. Feel free to ignore!